### ISSUE TITLE
CondaPkg dependency for Extension package

### ISSUE THREAD

**Issue by wsmoses** [JuliaPy/CondaPkg.jl#160]
I'm hoping to add a python dependency to an extension package, and it wasn't clear to me how this should be done.

Any tips?

x/ref https://github.com/EnzymeAD/Reactant.jl/issues/354 https://github.com/EnzymeAD/Reactant.jl/pull/407

cc @mofeing @avik-pal

**Comment by mofeing** [comment]
i guess that just adding `CondaPkg.toml` suffices? https://github.com/JuliaPy/PythonPlot.jl/blob/main/CondaPkg.toml

**Comment by wsmoses** [comment]
yeah the problem is we don't want to add the condapkg.toml on Reactant itself, but only when the PythonCall extension is loaded....and that....I have no idea how to do

**Comment by mofeing** [comment]
i guess that it'll still work? like CondaPkg.toml will only be checked when CondaPkg is loaded (or precompiled?)
